test_shopinvader_payment/tests/test_quotation.py

```diff
@@ -22,40 +21,47 @@ def test_get_no_cart_payment_info(self):
 class ShopinvaderQuotationPaymentCase(CommonConnectedQuotationCase):
     def setUp(self, *args, **kwargs):
         super(ShopinvaderQuotationPaymentCase, self).setUp(*args, **kwargs)
-        self.account_payment_mode = self.env.ref(
-            "shopinvader_payment.payment_method_check"
-        )
-        self.cart.write({"payment_mode_id": self.account_payment_mode.id})
+        self.payment_mode = self.env.ref(
+            "shopinvader_payment_manual.shopinvader_payment_check"
+        ).payment_mode_id
+        with self.work_on_services(
+            partner=self.partner, shopinvader_session=self.shopinvader_session
+        ) as work:
+            self.payment_service = work.component(usage="payment_manual")
 
     def test_get_cart_payment_info(self):
```

Comment:
It's difficult to understand the purpose of the test. We should improve the description by describing the data, the test case, the expected result

```python
def test_get_cart_payment_info(self):
    """
    Data:
        A sale order with a product requiring a quotation
    Test Case:
        Retrieve the quotation info
    Expected result:
        'available_methods' must be present into the informations
    """